recipes/shippinglabel/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "shippinglabel" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shippinglabel-{{ version }}.tar.gz
+  sha256: f78f8485769e2af157911314d1c853d65890d905ac8b2358a35957f780510db6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - apeye >=1.0.0
+    - dist-meta >=0.1.2
+    - dom-toml >=0.2.2
+    - domdf-python-tools >=3.1.0
+    - packaging >=20.9
+    - platformdirs >=2.3.0
+    - python >=3.6
+    - typing-extensions >=3.7.4.3
+
+test:
+  imports:
+    - shippinglabel
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/domdfcoding/shippinglabel
+  summary: Utilities for handling packages.
+  license: MIT
```

Looks like the trove license is apache 2.0:
```suggestion
  license: MIT AND Apache-2.0
```

---

Thanks. Wasn't quite sure what the syntax was for that.